components/About/data.tsx

```diff
@@ -199,7 +199,7 @@ export const phases = [
   },
   {
     status: Status.ComingSoon,
-    summary: 'Started May 10, 2022',
+    summary: 'Started May 10, 2022. Ending Nov. 15, 2022.',
```

Comment:
Not sure how I can force a newline in the framework, so truncated November instead.